packages/console/src/pages/SignInExperience/utils/form.ts

```diff
@@ -31,12 +32,16 @@ export const signInExperienceParser = {
     };
   },
   toLocalForm: (signInExperience: SignInExperience): SignInExperienceForm => {
-    const { signUp, signInMode } = signInExperience;
+    const { signUp, signInMode, customCss } = signInExperience;
 
     return {
       ...signInExperience,
       signUp: signInExperienceParser.toLocalSignUp(signUp),
       createAccountEnabled: signInMode !== SignInMode.SignIn,
+      // Specify default value in CodeEditor component wrapped by React Hook Form Controller can not properly displayed, hence use this workaround.
+      customCss: customCss?.length
+        ? customCss
+        : t('admin_console.sign_in_exp.custom_css.css_code_editor_content_placeholder'),
```

Comment:
would you like to paste a screenshot in the PR description?

Reply:
NP. Screenshot attached.